File: DSA/LakshaKundra_590013237/Day_44_Question_1.py
from queue import Queue

def display_queue(q):
    size = q.qsize()
    for _ in range(size):
        elem = q.get()
        print(elem, end=' ')
        q.put(elem)
    print()

q1 = Queue()
q1.put(10)
q1.put(20)
q1.put(30)
display_queue(q1)

q2 = Queue()
q2.put(5)
q2.put(15)
display_queue(q2)

File: DSA/PriyankaSingh_590016827/day44question1.c
#include <stdio.h>

#define SIZE 100

typedef struct {
    int data[SIZE];
    int front, rear;
} Queue;

void enqueue(Queue *q, int val) {
    q->data[q->rear++] = val;
}

void displayQueue(Queue q) {
    for (int i = q.front; i < q.rear; i++)
        printf("%d ", q.data[i]);
    printf("\n");
}

int main() {
    Queue q = {.front = 0, .rear = 0};
    enqueue(&q, 10);
    enqueue(&q, 20);
    enqueue(&q, 30);

    displayQueue(q);  // Output: 10 20 30

    return 0;
}
